Spigot-Server-Patches/0402-Add-more-Evoker-API.patch

```diff
@@ -0,0 +1,64 @@
+From 7ac07ac07ac07ac07ac07ac07ac07ac07ac07ac0 Mon Sep 17 00:00:00 2001
+From: BillyGalbreath <Blake.Galbreath@GMail.com>
+Date: Fri, 26 Oct 2018 02:39:29 -0500
+Subject: [PATCH] Add more Evoker API
+
+
+diff --git a/src/main/java/net/minecraft/server/EntityEvoker.java b/src/main/java/net/minecraft/server/EntityEvoker.java
+index 7ac07ac07ac0..7ac07ac07ac0 100644
+--- a/src/main/java/net/minecraft/server/EntityEvoker.java
++++ b/src/main/java/net/minecraft/server/EntityEvoker.java
+@@ -89,12 +89,13 @@ public class EntityEvoker extends EntityIllagerWizard {
+         return SoundEffects.ENTITY_EVOKER_HURT;
+     }
+ 
++    public void setWololoTarget(@Nullable EntitySheep sheep) { a(sheep); } // Paper - OBFHELPER
+     private void a(@Nullable EntitySheep entitysheep) {
+         this.c = entitysheep;
+     }
+ 
+-    @Nullable
+-    private EntitySheep dD() {
```

Comment:
Not a fan of moving the annotation around.

Reply:
It's an obf helper, it's absolutely required to or else wiggle will silently get it wrong (it's happened)